master_changes.md

```diff
@@ -618,3 +619,4 @@ users)
   * `OpamStd.Option`: add `equal` function [#5374 @rjbou]
   * `OpamStd.Compare`: add module to flag polymorphic comparison functions in opam codebase [#5374 @kit-ty-kate @rjbou]
   * `OpamStd.Env.`: introduce OpamStd.Env.Name to abstract environment variable names [#5356 @dra27]
+  * `OpamCompat`: Add `Fun.protect` [#5441 @kit-ty-kate]
```

Comment:
:heart:

Reply:
Hum we already had `OpamStd.Fun.finally` which is the same function (way before there was a `Fun` module in the stdlib :/)
It'd be best to ensure we consistently use the same function everywhere.